doc: nrf: bluetooth: RAS RREQ and RRSP docs (#19230)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

Bluetooth libraries and services
--------------------------------

> * :ref:`rreq_readme` library:
> 
>   * Added
> 
>     * Added the Ranging Service Ranging Requestor library for Channel Sounding.
> 
> * :ref:`rrsp_readme` library:
> 
>   * Added
> 
>     * Added the Ranging Service Ranging Responder library for Channel Sounding.
> 

Reviewer: ```suggestion
* Added the :ref:`rreq_readme` and :ref:`rrsp_readme` libraries.
```
Like this, but remove the changelog entry from this PR and add it as a suggestion to my https://github.com/nrfconnect/sdk-nrf/pull/19240.
